app/views/components/input_radio.scala.html

```diff
@@ -18,28 +18,32 @@
 
 @(
         field: Field,
-        legend: String,
-        legendClass: Option[String] = None,
-        hint: Option[String] = None,
+        labelHtml: Option[Html] = None,
+        contentHtml: Option[Html] = None,
+        hint: Option[Html] = None,
         trackGa: Boolean = false,
         inputs: Seq[RadioOption]
 )(implicit messages: Messages)
 
 <div class="form-group @if(field.hasErrors){form-group-error}">
     <fieldset id="@{field.id}">
         <legend>
-            <div class="bold-small @if(legendClass.nonEmpty){@{legendClass.get}}">@legend</div>
+            @if(labelHtml.isDefined){
+               @labelHtml
+            }
+            @if(contentHtml.nonEmpty){
```

Comment:
I'd prefer to see `isDefined` here. `nonEmpty` make it look like a collection.

You use `isDefined` on line 31 so to swap here is confusion IMO :)